### PR TITLE
Adding stdout console

### PR DESCRIPTION
**Pull request by zmhassan**
# Description 

Adding sink to output predictions to stdout and allow for users to click links in logs to give feedback right away.

### CODE REVIEW

File: anomaly_detector/storage/storage_catalog.py [hunk 1]
```diff
@@ -53,12 +54,19 @@ def _kafka_datasink_api(cls, config):
         logging.info("save kafka datasink")
         return KafkaSink(config=config)
 
+    @classmethod
+    def _stdout_datasink_api(cls, config):
+        """Kafka data sink."""
```
Comment (HumairAK) [inline anomaly_detector/storage/storage_catalog.py]: I think you forgot to edit this after copy pasting ;p

Comment (zmhassan) [reply anomaly_detector/storage/storage_catalog.py]: Fixed

**Comment by zmhassan** [comment]
@HumairAK Its all fixed.

**Comment by zmhassan** [comment]
Fixes: #326

**Pull request merged**